Demo/Demo/Extensions/PaymentButtonEnums+Extension.swift

```diff
@@ -69,7 +69,27 @@ extension PaymentButtonFundingSource: CaseIterable {
 extension PayPalButton.Label: CaseIterable {
     
     public static var allCases: [PayPalButton.Label] {
-        [.none, .checkout, .buyNow, .payWith]
+        [
```

Comment:
Can't we get `allCases` for free ([see sample](https://sarunw.com/posts/swift-caseiterable/#getting-all-cases-of-an-enum-with-swift-caseiterable))?

Reply:
Feel free to 🏈 punt on that one though - looks like an opportunity to cleanup our demo app in a follow-up PR

Reply:
Oh right, I overlooked using map filter and that would've made things easier... I'm going to open up a new PR for adding the optional tagline under the PayPal Button so I'll include that in there and demo app cleanup

Reply:
When I use
```